test/Microsoft.Buld.Sql.Tests/Microsoft.Build.Sql.Tests.csproj

```diff
@@ -1,19 +1,22 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFrameworks>net6.0</TargetFrameworks>
     <Nullable>enable</Nullable>
 
     <!-- Location to save the SDK Nuget Package. This will also serve as the local Nuget source when testing the sqlproj build. -->
     <SdkPackagePath>$(MSBuildThisFileDirectory)pkg\</SdkPackagePath>
+
+    <!-- Multi-targeting to test SDK on different .NET versions, requires all of the following SDKs to be installed -->
+    <TargetFrameworks Condition="'$(TestAllNetVersions)' == 'true'">$(TargetFrameworks);netcoreapp3.1;net5.0;net7.0</TargetFrameworks>
   </PropertyGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\..\src\Microsoft.Build.Sql\Microsoft.Build.Sql.csproj" />
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Build" Version="16.11.0" />
```

Comment:
Downgrading this a bit to support netcoreapp3.1, shouldn't be any issues since this is only for test